main/appserver/packager/glassfish-jcdi/pom.xml

```diff
@@ -95,6 +95,10 @@
             <artifactId>weld-osgi-bundle</artifactId>
         </dependency> 
         <dependency>
+            <groupId>org.jboss.classfilewriter</groupId>
```

Comment:
I didn't understand the need for classfilewriter changes. You removed the package exports from the weld-integration-fragment pom but bringing in dependency explicitly, why?